Generate API rewrite (#10)
snaplet/docs: docs/integrations/usage-with-prisma.md

# Usage with Prisma

> Prisma Migrations creates the database structure, and then Snaplet restores the data.

Reviewer: ```suggestion
Prisma Migrations creates the database structure, and Snaplet restores the data.
```

> Let Snaplet seed your database with Prisma by letting migrations create the database structure, and Snaplet restoring the data. 

Reviewer: 🤔 This sentence reads a bit weird now... Maybe something like:

```suggestion
Prisma Migrations creates the database structure, and then Snaplet restores the data.
```

```bash
prisma migrate dev
snaplet restore --data-only
```

> If you don't want to run two commands, you can automatically restore data, when running `prisma migrate reset` or `prisma migrate dev`, by adding the following to your `package.json` file.

Reviewer: ```suggestion
If you don't want to run two commands, you can automatically seed the database with data, after running `prisma migrate reset` or `prisma migrate dev`, by adding the following to your `package.json` file.
```
